_infra/helm/locust/locustfiles/requests.json

```diff
@@ -22,13 +58,13 @@
         },
         {
             "method": "GET",
-            "expected_response_text": "https://eq.surveys.onsdigital.uk/session?token=",
-            "response_status": 302,
             "harvest_url": {
                 "id": "surveyLongName",
-                "link_text": "Quarterly Business Survey",
-                "grouping": "/surveys/access-survey"
-            }
+                "link_text": "Quarterly Business Survey"
+            },
+            "expected_response_text": "session?token=",
+            "response_status": 302,
```

Comment:
`expected_response_status` ?

Reply:
Did it to avoid following 302's, mainly because of eq redirect. In our instance it's better to follow redirects as a general rule, but on some instances like this we don't want to. This gave me a leverage to do that